activation/activation_test.go

```diff
@@ -243,7 +243,7 @@ func (l *LayerClockMock) GetCurrentLayer() types.LayerID {
 func (l *LayerClockMock) AwaitLayer(types.LayerID) chan struct{} {
 	ch := make(chan struct{})
 	go func() {
-		time.Sleep(1 * time.Millisecond)
+		time.Sleep(1 * time.Second)
```

Comment:
For something that's flaky (not failing consistently), is 1000x increase really required? Maybe 50ms will be enough to make it stable?

Reply:
@noamnelke After running a few million iterations, I didn't see anything higher than 150ms. So, I rounded it up to the next power of 10. With lower values, the test can fail rarely, but still can.

Reply:
@noamnelke Do you think something like 200ms or 500ms is worth trying? Or doesn't it make much difference if we increase the timeout so much?

Reply:
This is not a timeout, but an actual sleep. A second isn't that much, so I leave it up to you, but based on what you say, maybe 300-500ms?

Reply:
Yes, something times out because of it, so I meant this sleep. 500ms seems to work fine for 10K iterations